### DISCUSSION TITLE
Chrome: Dragging a tab to create a new window does not trigger Rectangle when getting to the border

### DISCUSSION THREAD

**Discussion by olivierchatry** [rxhanson/Rectangle#552] [General]
- [ x ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: BigSur
Rectangle version: 0.48

When you drag a tab in chrome to create a new window, and, while dragging you try to drag it to a border, rectangle does not trigger. Might be a special case as I'm guessing Chrome have to do some weird thing to make that happen.

**Comment by rxhanson** [comment]
Indeed this is due to Chrome’s custom functionality here. Unfortunately, there is no way to implement this from Rectangle’s perspective.

**Comment by olivierchatry** [comment]
:( Wondering if there is anyway to catch this behavior maybe by spying what chrome is doing ? that might be a bit too involve.

**Comment by rxhanson** [comment]
As far as I know there is not. It might be possible to set up some kind of scripted solution with delays, like automatically executing keyboard shortcuts, but that would be outside of what I would desire to add into Rectangle.

**Comment by szferakarcolo** [comment]
This is very odd as on Windows this is working by default. :(

**Reply by olivierchatry** [reply]
Windows and OSX are two very different OS. Windows have Rectangle  functionalities embedded within it window manager. Rectangle have to hook inside OSX window manager to do what it does, so if an application does something a bit tricky, then it's hard or impossible to catch.
